### ISSUE TITLE
v3.0 beta BlockBreakEvent Exception

### ISSUE THREAD

**Issue by herm1ne** [WirelessRedstoneGroup/WirelessRedstone#170]
```
[23:11:03 ERROR]: Could not pass event BlockBreakEvent to WirelessRedstone v3.0.0-beta
org.bukkit.event.EventException
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:310) ~[spigot.jar:git-Spigot-21fe707-e1ebe52]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot.jar:git-Spigot-21fe707-e1ebe52]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at net.minecraft.server.v1_8_R3.PlayerInteractManager.breakBlock(PlayerInteractManager.java:286) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at net.minecraft.server.v1_8_R3.PlayerInteractManager.a(PlayerInteractManager.java:121) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:623) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at net.minecraft.server.v1_8_R3.PacketPlayInBlockDig.a(SourceFile:40) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at net.minecraft.server.v1_8_R3.PacketPlayInBlockDig.a(SourceFile:10) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_112]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_112]
        at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:715) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:374) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:654) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:557) [spigot.jar:git-Spigot-21fe707-e1ebe52]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_112]
Caused by: java.lang.NullPointerException
        at net.licks92.WirelessRedstone.Storage.SQLiteStorage.isChannelEmpty(SQLiteStorage.java:433) ~[?:?]
        at net.licks92.WirelessRedstone.Listeners.BlockListener.onBlockBreak(BlockListener.java:238) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor477.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_112]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_112]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot.jar:git-Spigot-21fe707-e1ebe52]
        ... 17 more
```
Reproduce Procedure: still under investigation; randomly happens when try to break signs (one of players confirmed, not me)

Did not lead to neither TPS drop nor crash, but I just want to report

**Comment by Bart-0110** [comment]
Hi, can you try this build? 

https://github.com/WirelessRedstoneGroup/WirelessRedstone/releases/tag/v3.0.0-beta.2

**Comment by herm1ne** [comment]
let me try this
but can you guess the way to reproduce? so that I can confirm the bug removal real quick!

**Comment by Bart-0110** [comment]
This should make a sign that WirelessRedstone recognizes with an invalid channel. It uses WorldEdit

```//set sign|[wrr]|invalidChannel|[default]```

**Comment by herm1ne** [comment]
fmm... the "[ ]" in the first line of sign [wrr]" is automatically removed if I made by using WorldEdit.
When I manually make the sign with the name invalidChannel, it does not throw exception!

However, I found a new bug
When I manually make a wrr sign, it says "You just created new channel!" then I right click on it,
the sign is automatically dropped and it says "This channel name contains invalid characters!"
BUT I activate the signal on the channel, the receiver sign is re-born and it turns to redstone torch.
The channel names I used are:  
test900
gp-testtest
